src/config.ts

```diff
@@ -293,9 +293,9 @@ export interface MarkConfig {
 
   // ---------- Point ----------
   /**
-   * The symbol shape to use. One of circle (default), square, cross, diamond, triangle-up, or triangle-down.
+   * The symbol shape to use. One of circle (default), square, cross, diamond, triangle-up, or triangle-down, or a custom SVG path.
    */
-  shape?: Shape;
+  shape?: string;
```

Comment:
Please use `Shape | string` so that enum still works and update docs `encoding.md` accordingly.